fxa-auth-db-server/test/backend/db_tests.js

```diff
@@ -291,8 +293,50 @@ module.exports = function(config, DB) {
                 t.deepEqual(token.emailCode, ACCOUNT.emailCode, 'token emailCode same as account emailCode')
                 t.equal(token.verifierSetAt, ACCOUNT.verifierSetAt, 'verifierSetAt is correct')
                 t.equal(token.accountCreatedAt, ACCOUNT.createdAt, 'accountCreatedAt is correct')
+                t.equal(token.tokenVerified, undefined, 'tokenVerified is undefined')
+                return db.sessionTokenVerified(SESSION_TOKEN_ID)
+              })
+              .then(function (token) {
+                t.deepEqual(token.tokenData, SESSION_TOKEN.data, 'token data matches')
+                t.deepEqual(token.uid, ACCOUNT.uid, 'token belongs to this account')
+                t.equal(token.createdAt, SESSION_TOKEN.createdAt, 'createdAt is correct')
+                t.equal(token.uaBrowser, 'foo', 'uaBrowser is correct')
+                t.equal(token.uaBrowserVersion, '1', 'uaBrowserVersion is correct')
+                t.equal(token.uaOS, 'bar', 'uaOS is correct')
+                t.equal(token.uaOSVersion, '2', 'uaOSVersion is correct')
+                t.equal(token.uaDeviceType, 'baz', 'uaDeviceType is correct')
+                t.equal(token.lastAccessTime, 42, 'lastAccessTime is correct')
+                t.equal(!!token.emailVerified, ACCOUNT.emailVerified, 'token emailVerified is same as account emailVerified')
+                t.equal(token.email, ACCOUNT.email, 'token email same as account email')
+                t.deepEqual(token.emailCode, ACCOUNT.emailCode, 'token emailCode same as account emailCode')
+                t.equal(token.verifierSetAt, ACCOUNT.verifierSetAt, 'verifierSetAt is correct')
+                t.equal(token.accountCreatedAt, ACCOUNT.createdAt, 'accountCreatedAt is correct')
+                t.equal(token.tokenVerified, 0, 'tokenVerified is 0')
+                return db.verifyToken(hex32(), { uid: ACCOUNT.uid })
+              })
+              .then(function() {
+                return db.sessionTokenVerified(SESSION_TOKEN_ID)
+              })
+              .then(function (token) {
+                t.equal(token.tokenVerified, 0, 'tokenVerified is 0')
+                return db.verifyToken(SESSION_TOKEN.tokenVerificationId, { uid: hex16() })
```

Comment:
It's not obvious at a glance why these are causing the `tokenVerified` to remain zero, it may be worth a comment saying e.g. `// Verify with wrong id, should not verify the token`.